tsl/src/build-defs.cmake

```diff
@@ -8,15 +8,20 @@ if (UNIX)
   set(CMAKE_C_FLAGS "${PG_CFLAGS} ${CMAKE_C_FLAGS}")
   set(CMAKE_CPP_FLAGS "${CMAKE_CPP_FLAGS} ${PG_CPPFLAGS}")
   set(CMAKE_C_FLAGS_DEBUG "${CMAKE_C_FLAGS_DEBUG} -g")
-endif (UNIX)
-
-if (APPLE)
+elseif(APPLE)
```

Comment:
I realize this PR is merged already, but this change doesn't look correct. 

UNIX used to apply to Mac before and APPLE just added Mac-specific options that shouldn't apply to UNIX in general. Now there seems to be a bunch of thing that are no longer set on mac.

Reply:
Ah, good point. Will look at that.